README.md

```diff
@@ -206,7 +206,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--skip-uncommitted-files-check` [default: false] - Skip check for uncommitted files in the analysis directory
     * `--upload` [default: false] - Request to push results to Codacy
     * `--upload-batch-size` [default: 50000] - Maximum number of results in each batch to upload to Codacy
-    * `--skip-docker-test` [default: false] - Skip the test_docker_socket().
+    * `--skip-docker-test` [default: false] - Skip testing if the default Unix socket for the Docker daemon exists
```

Comment:
Alternatively, the description could just mention testing if Docker is installed, but this would provide less information to allow the users to decide if they need the flag or not:

```suggestion
    * `--skip-docker-test` [default: false] - Skip testing if Docker is installed
```